components/SimpleCard/SimpleCard.tsx

```diff
@@ -27,8 +28,8 @@ export const SimpleCard = ({
   const isDarkBg = bgColor !== 'white' && bgColor !== 'fog-light';
 
   return (
-    <article className={cnb(style(hasLink, isDarkBg), allCardBgColors[bgColor], className)} {...props}>
+    <div className={cnb(style(hasLink, isDarkBg), allCardBgColors[bgColor], className)} {...props}>
       {children}
-    </article>
+    </div>
```

Comment:
Basic card is used for a lot of random things so `article` tag isn't always appropriate